bin/ch-test

```diff
@@ -88,6 +88,8 @@ EOF
 
 bats_vmin=1.2.0  # FIXME: get from configure somehow?
 
+ch_build_unpack_all="argenv quick copy almalinux_8ch debian_11ch hello multistage seccomp"
```

Comment:
Should derive this list from the tests we actually have; it should also be responsive to `--scope`.